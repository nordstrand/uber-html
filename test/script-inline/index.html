<!DOCTYPE html>
<html>
<head>
    <title>test</title>
</head>
<body>
    <script src="js/a.js"></script>
    <script>console.log('dont touch me')</script>
</body>
</html>