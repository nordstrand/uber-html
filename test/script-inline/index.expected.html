<!DOCTYPE html>
<html>
<head>
    <title>test</title>
</head>
<body>
    <script>console.log("doh");</script>
    <script>console.log('dont touch me')</script>
</body>
</html>